image-loader/test/scala/lib/imaging/FileMetadataReaderTest.scala

```diff
@@ -558,7 +558,7 @@ class FileMetadataReaderTest extends FunSpec with Matchers with ScalaFutures {
       "Custom Rendered" -> "Normal process",
       "Exif Image Height" -> "3840 pixels",
       "Lens Serial Number" -> "000043c4bb",
-      "Flash" -> "Flash did not fire, auto",
+      "Flash" -> "Flash did not fire",
```

Comment:
metadata-extractor appear to have updated their [canned response for flash info](https://github.com/drewnoakes/metadata-extractor/search?q=%22Flash+did+not+fire%22&unscoped_q=%22Flash+did+not+fire%22)